### PR TITLE
Issue to review the contributing document and add a triage process proposal

### PR DESCRIPTION
**Pull request by benjagm**
**Summary**
**GitHub Issue:** #583 
- Review the Readme or equivalent.
- The main place with the link to contribute.
- Repo shield badges.
- Review the contributing guide or equivalent and improve if necessary
- Review triage
- All contributors are visible and aligned
- Review all the places where this contribution path is enabled and make it consistent
- Document the role of Community Manager

**Do you think resolving this issue might require an [Architectural Decision Record (ADR)](https://github.com/json-schema-org/community/blob/main/CONTRIBUTING.md#key-architectural-decisions)? (significant or noteworthy)**

No

### CODE REVIEW

**Comment by Relequestual** [comment]
We currently use a modified [Sensible Github Labels](https://medium.com/@dave_lunny/sane-github-labels-c5d2e6004b63) list of labels: https://github.com/Relequestual/sensible-github-labels. I'd like to continue to use that as the basis, and not have similar/overlapping labels to keep it consistent across the org.

Broadly though, this looks good.

I'd like to have all new issues have a `triage` label.

I feel we should use all the severity labels defined. I think "priority" makes better sense given it's not nesecerily an issue of severity.

I think all questions should be moved to Discussions.

Whatever we end up with, we also need to make sure it is applied to all active repos, which includes making sure they have the labels and an updated readme, which may just direct to this document.

File: TRIAGE.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+| `High priority`          | High priority flow or feature is broken/non-functional, crash.            |
+| `Critical priority`      | Indicates Critical impact on data or site: data loss (posts, pages, comments), unexpected publishing, security issue.|
+
+
+
+Other labels:
+| Label                    | Description                                                               |
+| ------------------------ | ------------------------------------------------------------------------- |
+| `good first issue`       | Indicates that the issue is suitable for newcomers to contribute to the project.|
+| `question`               | Indicates that the issue lacks sufficient information to be able to act on it right now and requires additional details from the reporter. The maintainer for the area must remove the tag when more info is provided, or close it if it is too old.|
```
Comment (Relequestual) [inline TRIAGE.md:41]: I don't think `question` is the right label here. Where we need more infomation, we should use something like `info-required`. I expect Issues which are questions will be moved to Discussions.

File: TRIAGE.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+1. Make sure to search in previous issue to ensure this is not a duplicated issue. If that is the case, mark it as `duplicate` and close it referencing the original issue. If the issue is clear, ensure it has the correct `area` assigned. Also consider if it is a `good first issue`.  
+
+2. Asses the issue severity to determine what functionality is affected and how broken it is and use the severity labels just for High and Critical cases.
+
+3. If the submitter or the reviewer believe the resolution of the Issue is likely to result in a significant or noteworthy change it will be labeled as `adr-required`.
+
+4. Move the issue to the correct repo if required.
+
+5. If it's not clear what the problem or issue is, add a note for the requestor (or occasionally some other user on the thread), and mark it as `question`. If a `question` issue persists for longer than a week or so (past a second followup), it's reasonable to `close` the issue and encourage the requester to reopen if they have more details.
```
Comment (Relequestual) [inline TRIAGE.md:56]: I think this needs revision as per my other comments.

**Review by Relequestual — CHANGES_REQUESTED** [review]
I think this is a great first start, which puts us in a much better place than previous. Let's re-work per my comments. (Obviously, more comments and reviews welcome.)

**Comment by benjagm** [comment]
> We currently use a modified [Sensible Github Labels](https://medium.com/@dave_lunny/sane-github-labels-c5d2e6004b63) list of labels

Thanks @Relequestual . Sensible GitHub labels are pretty flexible and cover most use cases. Is there any place where we have documented what is the modification we are using?

**Comment by benjagm** [comment]
> Let's re-work per my comments.

Please take a look to the new version I just pushed. Hope it reflects all the feedback provided.

File: TRIAGE.md (hunk 2 above)
Comment (benjagm) [reply TRIAGE.md:73]: This is fixed in the new version of the PR.

**Comment by benjagm** [comment]
There is a new version of the PR ready addressing all the comments received.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -1,16 +1,37 @@
...
 
-The fastest way to get involved and become active in the community is to join our Slack server at https://json-schema.org/slack.
+The fastest way to get involved and become active in the community is to join the `#contribute` channel in our Slack server at https://json-schema.org/slack.
 
 If you have suggestions on how to improve our community, please do check out our [Community Discussions](https://github.com/json-schema-org/community/discussions), and start a new discussion.
 
 # General
 
-While each repository may have its own guidance for contributing, the following sections apply organisation wide.
+The JSON Schema organization hosts different projects, and while most have similar  contributing guidelines as described in this document, the JSON Schema Specification development process follows different contributing guidelines. Please, check-out the [Spec repository](https://github.com/json-schema-org/json-schema-spec/blob/main/CONTRIBUTING.md) to know more.
```
Comment (Relequestual) [inline CONTRIBUTING.md:11]:
```suggestion
The JSON Schema organization hosts different projects, and while most have similar contributing guidelines as described in this document, the JSON Schema Specification development process follows different contributing guidelines. Please, check-out the [Spec repository](https://github.com/json-schema-org/json-schema-spec/blob/main/CONTRIBUTING.md) to know more. 
```

**Review by Relequestual — APPROVED** [review]
Looks great! Thanks for doing this work!

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -26,12 +20,15 @@ Here are some areas where you can contribute:
...
+ * 📋 [Improving The Documentation](https://github.com/orgs/json-schema-org/projects/16)
+ * 🏗 [Website development](https://github.com/orgs/json-schema-org/projects/11/views/1)
  * 🎨 [Improving The Design](https://github.com/json-schema-org/website/blob/main/CONTRIBUTING.md#-improving-the-design)
- * 😍 [Collaborating with Community Management tasks](https://github.com/json-schema-org/community)
+ * 😍 [Collaborating with Community Management tasks](#community-management-tasks)
  * 🚀 [Adding a new Adopter](https://github.com/json-schema-org/community/blob/main/ADOPTERS.md)
 
+#### Community management tasks
+
+To get involved in the Community management tasks, please join the `#community-mgmt` slack channel using this [link](#community-mgmt).
```
Comment (Relequestual) [inline CONTRIBUTING.md:31]: Does the link here need adding?

```suggestion
To get involved in the Community management tasks, please join the [`#community-mgmt` slack channel](https://json-schema.slack.com/archives/C0209BTU2UQ).
```

File: README.md [hunk 5]
```diff
@@ -1,48 +1,97 @@
...
+JSON Schema is a community that lives on its contributors. As we grow, we need more people to help others. Do you like to get involved? You can contribute in many ways!
 
-# Who decided all this? Is it up for debate?
+Please check out our [Contribution guidelines](CONTRIBUTING.md) to know more.
 
-Ben Hutton (@relequestual) created this repo and initial readme on April 2021, based on the expectations of community and organisational needs having grown, supported, and observed the community over the past 5 years.
+### Contributors
 
-**None of this is set in stone. Humans are fallible. Everything is up for debate.**
+Thanks goes to these wonderful people who contributed to this website:
```
Comment (Relequestual) [inline README.md:89]:
```suggestion
Thanks goes to these wonderful people who contributed to this communtiy space:
```

**Review by Relequestual — CHANGES_REQUESTED** [review]
Two very minor suggestions, otherwise it looks great!
Thank you for your work on this. Looks so much better.
If you agree with the suggestions, please feel free to merge after accepting them 🥳

**Comment by benjagm** [comment]
> If you agree with the suggestions, please feel free to merge after accepting them 🥳

Thanks for the review and the suggestions @Relequestual !! Let's merge it!

**Pull request merged**